package.json

```diff
@@ -65,7 +65,7 @@
     "generate-scaffolding": "repo-tools generate all && repo-tools generate lib_samples_readme -l samples/ --config ../.cloud-repo-tools.json"
   },
   "dependencies": {
-    "@google-cloud/common": "^0.19.2",
+    "@google-cloud/common": "^0.20.0",
```

Comment:
This is unrelated to dropping `node4` support and should go to a separate PR.

Reply:
And if you don't make this change here, you can safely revert your lock files as lock files are supposed to be changed only if package dependencies are changed :)